Add mk47 keyboard. (#19797)
qmk/qmk_firmware: keyboards/hfdkb/keyboard_sw/mk47/readme.md

# mk47

A customizable 40% keyboard.

* Keyboard Maintainer: [jonylee@hfd](https://github.com/jonylee1986)

> * Hardware Supported: mk47

Reviewer: ```suggestion
* Hardware Supported: mk47 PCB w/ WB32FQ95
```

Make example for this keyboard (after setting up your build environment):

    make hfdkb/keyboard_sw/mk47:default

> 

Reviewer: ```suggestion

Flashing example for this keyboard:

    make hfdkb/keyboard_sw/mk47:default:flash
```

# mk47

A customizable 40% keyboard.

* Keyboard Maintainer: [jonylee@hfd](https://github.com/jonylee1986)
* Hardware Supported: mk47 PCB w/ WB32FQ95

Make example for this keyboard (after setting up your build environment):

    make hfdkb/keyboard_sw/mk47:default

Flashing example for this keyboard:

    make hfdkb/keyboard_sw/mk47:default:flash

> **Reset Key**: Hold down the key located at *K01*, which programmed as *Esc* while plugging in the keyboard.

Reviewer 2: ```suggestion
**Reset Key**: Hold down the key top-left key (default *Esc*) while plugging in the keyboard.
```
